src/Controls/tests/TestCases.Shared.Tests/Tests/Issues/Issue24878.cs

```diff
@@ -0,0 +1,23 @@
+﻿#if ANDROID
```

Comment:
Is it relevant to run this test on all platforms?

Reply:
I enabled it on Windows